script/constructron.lua

```diff
@@ -730,7 +657,8 @@ me.actions = {
 
     ---@param _ LuaEntity[]
     ---@param chunk Chunk
-    check_decon_chunk = function(_, chunk)
+    check_decon_chunk = function(job, chunk)

```

Comment:
annotations are not correct here anymore. `_` is gone and `job` is new.

Reply:
I added this in a the next commit